config/opal_check_alt_short_float.m4

```diff
@@ -1,9 +1,7 @@
 dnl -*- shell-script -*-
 dnl
-dnl Copyright (c) 2018-2020 FUJITSU LIMITED.  All rights reserved.
+dnl Copyright (c) 2018-2021 FUJITSU LIMITED.  All rights reserved.
 dnl Copyright (c) 2020 Cisco Systems, Inc.  All rights reserved.
-dnl Copyright (c) 2021      Triad National Security, LLC. All rights
-dnl                         reserved.
```

Comment:
Did you mean to remove a copyright here?

Reply:
@jsquyres I reverted 9e2d9bd and then made a new change. The copyright was included in 9e2d9bd.